apps/bfd-pipeline/bfd-pipeline-rif-extract/src/test/java/gov/cms/bfd/pipeline/rif/extract/RifFilesProcessorTest.java

```diff
@@ -105,6 +105,70 @@ public void process1BeneRecord() {
     Assert.assertEquals(new BigDecimal("1"), beneRow.getBeneLinkKey().get());
   }
 
+  @Test
+  public void process1BeneRecordWithTrailingNulls() {
```

Comment:
I'd probably simplify this way down by:
* Checking any/all fields with null bytes, to ensure they were cleaned up correctly.
* Checking one field _after_ the null byte(s), to ensure they're correct.
* Getting rid of most other assertions.

As-is, it's a bit hard to figure out which field(s) were supposed to be corrected. Maybe also add a comment mentioning that?

Reply:
Done